modules/installation-gcp-config-yaml.adoc

```diff
@@ -37,21 +37,13 @@ controlPlane: <2>
   name: master
   platform:
     gcp:
-      rootVolume:
-        iops: 4000
-        size: 500
-        type: io1
       type: n1-standard-4
```

Comment:
GCP allows user to modify zones hence suggesting to add zones, e.g.
platform:
  gcp:
    type: n1-standard-4
    zones: us-central1-a